protocol_rfcs/redirection.md

```diff
@@ -0,0 +1,116 @@
+# Redirection
```

Comment:
Please update the table of proposed RFCs - https://github.com/delta-io/delta/tree/master/protocol_rfcs#table-of-rfcs

in fact, i realized that the instructions in that protocol_rfcs/README does not say to update the table. Can you please add that.

Reply:
Done